src/test/python/parser/opentrackio_parser.py

```diff
@@ -52,14 +52,13 @@ def main():
         print("Usage: python3 opentrackio_parser.py --file=opentrackio_sample.json --schema=opentrackio_schema.json")
         exit (-1)
 
-    sample = OpenTrackIOProtocol(sample_text,schematext,verbose) # a "Sample" is a de-serialized JSON object containing the protocol
-    sample.parse()                # parse the actual JSON of the protocol
-    sample.import_schema()        # read the schema which governs the interpretation of the protocol
```

Comment:
import_schema() went away in your code, but is schema checked inside the library on-the-fly now.  If so thats OK. Without knowing default units the unit conversions are not possible of course.

Reply:
Yes, it became a little cleaner to create a new instance that way.